File: nx-time-backend/src/main/java/com/nxtime/nxtime/NxTimeBackendApplication.java
package com.nxtime.nxtime;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableScheduling;

/**
 * @EnableScheduling desde la Fase 9: activa el proceso nocturno que
 * cierra las jornadas que nadie cerró (ver IncompleteTimeEntryScheduler).
 */
@SpringBootApplication
@EnableScheduling
public class NxTimeBackendApplication {

    public static void main(String[] args) {
        SpringApplication.run(NxTimeBackendApplication.class, args);
    }
}

File: nx-time-backend/src/main/kotlin/com/nxtime/nxtime/NxTimeBackendApplication.kt
package com.nxtime.nxtime

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication


@SpringBootApplication
class NxTimeBackendApplication

/*
 * Esta es la función principal que arranca toda la aplicación de backend.
 */
fun main(args: Array<String>) {

    runApplication<NxTimeBackendApplication>(*args)
}
